### ISSUE TITLE
Feature request - The unfollow 10, then wait 10 min

### ISSUE THREAD

**Issue by jeffreylm** [instabot-py/instabot.py#675]
Hello,

One of the things that Instagram has is that you can only unfollow in sets of 10 and if you do more than that, they will block you from unfollowing for 10 minutes. 

**Feature**

- Unfollow 10 people then wait ten minutes then repeat.
- Within that ten minute span of waiting, continue doing the rest of the script and return after 10 minutes to unfollow another 10 people.
- Unfollow 10 people at a random speed, but more close to human speed as to not raise a red flag by instagram. The unfollow (and also the follow) needs to seem as though a human is doing it. So in this case slower is better. At the moment your script does a rapid fire with those two features. It is only in a matter of time that this will raise a red flag.

There is also one other feature that i would appreciate is a space between each executed block, makes it more readable.

Kind regards,
Jeffrey

**Comment by TehJokeR** [comment]
Just use mode 0, it works better than this. It will unfollow exactly how many people you want to unfollow per day. You cant beat the system by only unfollowing every 10 minutes, there's still a cap on the max amount of follows per day so your account will still be flagged.

Also the amount of unfollows you can do at any single time is dependent on how long you have your account and how 'built' it is. i.e followers/followings. I can unfollow 500 people in a row if I want, mine doesnt cap at 10.

**Comment by jeffreylm** [comment]
Thank you for that explanation. I am also able to unfollow 500 in a row with no problem. From my understanding there is a max of 1000 unfollows per day. And what about mode 2. Would you also advize that based on my above? 

Like that it will also take out the unfollowers. Find that they have the highest priority to get unfollowed.

Greetings,
Jeffrey

**Comment by TehJokeR** [comment]
At the start I created my own mode and tried to do things like you suggested. It also slept between 10pm and 8am every day to make it look authentic, but then I just switched to Mode 0 and began using it and it works perfect. I'm unsure about mode 2, I would only recommend mode 1 as this is the baseline code that everything else was built on and it works perfect as described in the README. The other modes are different flavors of the bot, so might not match exactly what it says it does in the README
